api/overview/azure/latest/extensions.aspnetcore.configuration.secrets-readme.md

```diff
@@ -5,10 +5,9 @@ author: jsquire
 ms.author: jsquire
 ms.date: 04/06/2022
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: dotnet
-ms.service: extensions
+ms.service: key-vault
```

Comment:
This should be updated in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv